Content.Shared/Language/Components/LanguageSpeakerComponent.cs

```diff
@@ -10,23 +11,35 @@ namespace Content.Shared.Language;
 ///     All fields of this component are populated during a DetermineEntityLanguagesEvent.
 ///     They are not to be modified externally.
 /// </remarks>
-[RegisterComponent]
+[RegisterComponent, NetworkedComponent]
 public sealed partial class LanguageSpeakerComponent : Component
 {
+    public override bool SendOnlyToOwner => true;
+
     /// <summary>
     ///     The current language the entity uses when speaking.
     ///     Other listeners will hear the entity speak in this language.
     /// </summary>
     [DataField]
-    public string CurrentLanguage = ""; // The language system will override it on init  
+    public ProtoId<LanguagePrototype> CurrentLanguage = ""; // The language system will override it on init
 
     /// <summary>
     ///     List of languages this entity can speak at the current moment.
     /// </summary>
-    public List<string> SpokenLanguages = [];
+    [DataField]
+    public List<ProtoId<LanguagePrototype>> SpokenLanguages = [];
```

Comment:
I don't really see the benefit, but I see how it would be highly inconvenient for every caller to check those fields for null, whereas in reality they would never be null. The performance difference should be very negligible here, although if it concerns you a lot, I simply can change the initializer to `default!`, without changing the field type.